helpers/form/class.FormElement.php

```diff
@@ -21,10 +21,12 @@
  */
 
 use oat\oatbox\validator\ValidatorInterface;
+use oat\tao\helpers\form\elements\xhtml\SearchDropdown;
 use oat\tao\helpers\form\elements\xhtml\SearchTextBox;
 
 // Defining aliases for old style class names for backward compatibility
-class_alias(SearchTextBox::class, \tao_helpers_form_elements_xhtml_Searchtextbox::class);
+class_alias(SearchTextBox::class, tao_helpers_form_elements_xhtml_Searchtextbox::class);
```

Comment:
Be careful with removing root namespace definition via `\`, I had issues when by default it was resolved in the namespace that defined in the file, and the class was not found.
```suggestion
class_alias(SearchTextBox::class, \tao_helpers_form_elements_xhtml_Searchtextbox::class);
```

Reply:
Right, makes sense, if a class name is to be used in a namespaced context.